src/components/attribute-picker/attribute-picker.js

```diff
@@ -426,10 +426,18 @@ class AttributePicker extends ArrowKeysMixin(LocalizeLabsElement(LitElement)) {
 	}
 
 	_onInputTextChanged(e) {
+		e.stopPropagation();
```

Comment:
Right now, the `input` event from the inner `<input>` element bubbles up through the DOM and can be caught by consumers of `<d2l-labs-attribute-picker>`. This event has `target` set to `<d2l-labs-attribute-picker>`. Since `<d2l-labs-attribute-picker>` doesn't have a `value` property, `event.target.value` ends up being `undefined`.

As such, to discourage listening to the `input` event and encourage listening to `d2l-attribute-picker-text-changed` instead, the `input` event's propagation is stopped here.

Reply:
If you suppress the `input` event, existing consumers [like this one](https://github.com/Brightspace/manager-view-fra/blob/2f258ffa9a50820b23312135455aa65ef089ecd0/src/group/mr-group-membership-rules.js#L261) are going to stop working I think.

Reply:
I see, thanks for catching that. I'll remove this line then to avoid breakage. I suppose existing consumers can make the decision to switch over to the new event as they see fit.